src/ProxyClient/Varnish.php

```diff
@@ -36,7 +36,10 @@ public function __construct(
     private function fetchAndMergeAuthHeaders(array $headers): array
     {
         if ($this->configResolver->hasParameter('http_cache.varnish_invalidate_token')) {
-            $headers[InvalidateTokenController::TOKEN_HEADER_NAME] = $this->configResolver->getParameter('http_cache.varnish_invalidate_token');
+            $token = $this->configResolver->getParameter('http_cache.varnish_invalidate_token');
+            if ('' !== $token) {
+                $headers[InvalidateTokenController::TOKEN_HEADER_NAME] = $token;
```

Comment:
```suggestion
            $purgeAuthToken = $this->configResolver->getParameter('http_cache.varnish_invalidate_token');
            if ('' !== $purgeAuthToken) {
                $headers[InvalidateTokenController::TOKEN_HEADER_NAME] = $purgeAuthToken;
```

Or something similar to make it more clear what it is.